lib/watir-webdriver/locators/element_locator.rb

```diff
@@ -233,6 +242,21 @@ def delete_regexps_from(selector)
       rx_selector
     end
 
+    def regexp_selector_to_predicates(key, re)
+      return [] if re.casefold?
```

Comment:
Originally I did that but then realized `XpathSupport.downcase` only works for ASCII characters. Since this is an optimization, I didn't want to tackle too many edge cases at the risk of introducing bugs.